pdpserver/Makefile

```diff
@@ -10,9 +21,8 @@ install:
 	go install ./... 
 
 .PHONY: lint
-lint:
-	go get -v github.com/golang/lint/golint
-	if [ -n "$$(golint *.go)" ]; then \
+lint: $(GOLINT)
+	if [ -n "$$(golint ./...)" ]; then \
```

Comment:
I played a bit with the command and found that following formula prints output and returns exit status:
```make
GOLINTDIAG=$$($(GOLINT) ./...)`; [ -z "$$GOLINTDIAG" ] || (echo "$$GOLINTDIAG" && exit 1)
```
but then I read help for golint and found that it has all we need option - "-set_exit_status". So we just need:
```make
.PHONY: lint
lint: $(GOLINT)
	$(GOLINT) -set_exit_status ./...
```